clients/src/test/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManagerTest.java

```diff
@@ -152,6 +152,34 @@ public void cleanup() {
         }
     }
 
+    @Test
+    public void testHeartBeatRequestStateToStringBase() {
+        final long retryBackoffMs = 100;
+        final long retryBackoffMaxMs = 1000;
+        LogContext logContext = new LogContext();
+        HeartbeatRequestState heartbeatRequestState1 = new HeartbeatRequestState(
```

Comment:
Super nit: we can drop the `1` at the end of the variable name, right?

```suggestion
        HeartbeatRequestState heartbeatRequestState = new HeartbeatRequestState(
```